packages/orbit-components/src/Itinerary/Itinerary.ct.tsx

```diff
@@ -59,7 +59,7 @@ test.describe("visual Itinerary", () => {
   test("ItinerarySegment open", async ({ mount }) => {
     const component = await mount(<ItinerarySegmentVisualStory />);
 
-    await component.getByTestId("segment").click();
+    await component.getByText("PRG").click();
```

Comment:
I assume this is related to this statement

> The visual test click selector was changed because, by coincidence, the click was being applied precisely on the HorizontalScroll element


I'm curious, how did you know it was clicking on the HorizontalScroll component? 👀